lib/rubocop/cop/rspec/spec_file_path_format.rb

```diff
@@ -57,6 +66,28 @@ def on_top_level_example_group(node)
           end
         end
 
+        # For testing and debugging
+        def self.reset_activesupport_cache!
+          @activesupport_available = nil
+        end
+
+        def self.activesupport_inflections_available?(inflections)
+          return @activesupport_available unless @activesupport_available.nil?
+
+          @activesupport_available = begin
+            return false unless File.exist?(inflections)
+
+            require 'active_support/inflector'
+            require inflections
+
+            true
+          rescue LoadError, NameError
+            false
+          end
+        rescue StandardError
```

Comment:
this separation in error handling is required by the Shadowing rule

Reply:
Yes, `NameError` is a subclass of `StandardError`. I think you can just change line 84 to `rescue LoadError, StandardError`.

Reply:
Fixed!